src/test/resources/no/skatteetaten/aurora/boober/facade/DeployFacadeTest/python-spec-default.txt

```diff
@@ -71,7 +71,10 @@ utv/python.json | sts: true
                 |   jwtToStsConverter:
         default |     enabled: false
         default |     version: "0.4.0"
+        default |     discoveryUrl: "http://login-microsoftonline-com.app2ext.sikker-prod.skead.no/common/discovery/keys"
         default |     ivGroupsRequired: false
+        default |     ldapUserVaultName: ""
+        default |     ldapUrl: "ldap://ldap.skead.no:389"
```

Comment:
Kunne vi brukt dummy-verdier for alle disse url'ene?

Reply:
Dette løses av https://github.com/Skatteetaten/boober/pull/547#discussion_r873398838